### ISSUE TITLE
New folder is created unpaused when "Pause all" was selected

### ISSUE THREAD

**Issue by bugith** [syncthing/syncthing#4002]
v0.14.24-rc.1
New folder in that case should be created paused. This would be a way to workaround #3608 and #3996

**Comment by AudriusButkevicius** [comment]
No, I disagree, the button means pause all existing folders.

**Comment by bugith** [comment]
Haaa ? You'd rather say it's the way it works, not what it means. Then the button should be labelled "Pause all existing"/"Resume all above" isn't it ? ;-)

**Comment by AudriusButkevicius** [comment]
No, I am saying what the button does. It's not like the button sticks or anything, it's a one off action.

**Comment by calmh** [comment]
I don't particularly care but I'd be fine with setting the new folder state to the majority of existing folder states.

**Comment by AudriusButkevicius** [comment]
It's not intuitive to me tbh, but yeah it could work.

**Comment by bugith** [comment]
> setting the new folder state to _the majority_ of existing folder states
Oooch, complicated, I believed there was already existing some global "All_Paused_Status=0|1" flag.

**Comment by wweich** [comment]
How about using a split button for *Save*, where one of the additional options is *Save paused* or *Save & Pause*.

http://getbootstrap.com/components/#btn-dropdowns-split

**Comment by uok** [comment]
Keep it simple! I'm with @AudriusButkevicius on this one. If special settings for new folder are needed just create it and don't share it, at least that's how I'd do it

**Comment by calmh** [comment]
I would expect that in the vast majority of cases all folders will be unpaused, and then so will the new one. When all folders are paused the new one becomes paused. The mixtures are probably more unusual and the user can always "eh what?" and fix it if it was unexpected. When it's fifty fifty we should go with not paused.

**Comment by bugith** [comment]
Don't worry guy. We already have several way uok says. The only thing I had in mind when I posted this not-a-bug was when we need to set some ignores. And anyway in this case we need to enter back in edit mode to set the ignores, so it is really easy to select the pairs at this moment.
My concern was only for new users that don't yet have this idea to set the ignores before they select pairs.
Still sad for some days until time does its job, as I buried my dog yesterday. Have a good day all and thanks again for your kind job.
